### ISSUE TITLE
How to bypass usage of mask and get full rendered image in evaluation

### ISSUE THREAD

**Issue by InduCherukuri** [TencentARC/ArcNerf#2]
I have a couple of questions regarding the usage of mask:

1. Is it used to reduce the memory load?
2. How do I bypass this and get the fully rendered image in evaluation instead of masked output?

I have changed the mask at, multiple places and I'm unable to bypass it. I have also changed nerf_model.py to give full output by adding the following line
mask_pts = torch.where(mask_pts==True, False, False)
but I'm getting an error

<img width="1509" alt="error1" src="https://user-images.githubusercontent.com/20127155/227443455-dfc576cc-3c7b-430b-ba4b-ea610b540368.png">

_Originally posted by @InduCherukuri in https://github.com/TencentARC/ArcNerf/issues/1#issuecomment-1482319919_
